src/dates/metar.ts

```diff
@@ -8,6 +9,21 @@ export interface IMetarDated extends IMetar {
 export function metarDatesHydrator(report: IMetar, date: Date): IMetarDated {
   return {
     ...report,
+    remarks: report.remarks.map((remark) => {
```

Comment:
Do real METARs in Canada have NXT FCST BY?

If not, it is ok to not hydrate here since it should never happen (if it does, lets add a test case for coverage)

Reply:
No its only something TAF has I added it for completion. Gonna remove it.